zsh/_brew-koji

```diff
@@ -43,482 +121,1268 @@ _brew-koji() {
+            '*: name' \ 
+            && ret=0 
+            ;;
+       (edit-tag-inheritance)
+         _arguments -A "-*" \
+            '(--priority --maxdepth --intransitive --noconfig --pkg-filter )'{-h,--help}'[show this help message and exit]' \
+            '--priority=-[Specifya new priority]' \ 
+            '--maxdepth=-[Specify max depth]' \ 
+            '--intransitive[Set intransitive]' \ 
+            '--noconfig[Set to packages only]' \ 
+            '--pkg-filter=-[Specify the package filter]' \ 
+            '*: tag <parent> <priority>' \ 
+            && ret=0 
+            ;;
+       (edit-target)
+         _arguments -A "-*" \
+            '(--rename --build-tag --dest-tag )'{-h,--help}'[show this help message and exit]' \
+            '--rename=-[Specify new name for target]' \ 
+            '--build-tag=-[Specify a different build tag]' \ 
+            '--dest-tag=-[Specify a different destination tag]' \ 
+            '*: name' \ 
+            && ret=0 
+            ;;
+       (edit-user)
+         _arguments -A "-*" \
+            '(--rename --edit-krb --add-krb --remove-krb )'{-h,--help}'[show this help message and exit]' \
+            '--rename=-[Rename the user]' \ 
+            '--edit-krb=-[Change kerberos principal of the user]' \ 
+            '--add-krb=-[Add kerberos principal of the user]' \ 
+            '--remove-krb=-[Remove kerberos principal of the user]' \ 
+            '*:name ' \ 
+            && ret=0 
+            ;;
+       (enable-host)
+         _arguments -A "-*" \
+            '(--comment )'{-h,--help}'[show this help message and exit]' \
+            '--comment=-[Comment indicating why the host(s) are being enabled]' \ 
+            '*: hostname ...' \ 
+            && ret=0 
+            ;;
+       (enable-user)
+         _arguments -A "-*" \
+            '()'{-h,--help}'[show this help message and exit]' \
+            '*:username' \ 
+            && ret=0 
+            ;;
+       (free-task)
+         _arguments -A "-*" \
+            '()'{-h,--help}'[show this help message and exit]' \
+            '*: <task-id> [<task-id> ...]' \ 
+            && ret=0 
+            ;;
+       (grant-cg-access)
+         _arguments -A "-*" \
+            '(--new )'{-h,--help}'[show this help message and exit]' \
+            '--new[Create a new content generator]' \ 
+            '*:<user> <content generator>' \ 
+            && ret=0 
+            ;;
+       (grant-permission)
+         _arguments -A "-*" \
+            '(--new )'{-h,--help}'[show this help message and exit]' \
+            '--new[Create a new permission]' \ 
+            '*:<permission> <user> [<user> ...]' \ 
+            && ret=0 
+            ;;
+       (import)
+         _arguments -A "-*" \
+            '(--link --test --create-build --src-epoch )'{-h,--help}'[show this help message and exit]' \
+            '--link[Attempt to hardlink instead of uploading]' \ 
+            '--test[Don't actually import]' \ 
```

Comment:
The `Don't` causes quoting issues. I think in this causes its better to change all instances to

```
"--test[Don't actually import]" \
```

Note that there are also a few instances of `Dont` that could be fixed up as well.

Reply:
Fixed all `Don't` instances:

$ grep -i "dont" brew-koji/zsh/_brew-koji 
$ grep -i "don't" brew-koji/zsh/_brew-koji
            "--test[Don't actually import]" \
            "--test[Don't actually import]" \
            "--test[Test mode -- don't actually import]" \
            "--nowait[Don't wait on for regen to finish]" \
            "--nowait[Don't wait on task]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on image creation]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on build]" \
            "--nowait[Don't wait on task]" \
            "--nowait[Don't wait on appliance creation]" \
            "--nowait[Don't wait on livecd creation]" \
            "--nowait[Don't wait on livemedia creation]" \
            "--nowait[Don't wait on build]" \
            "--skip-tag[If creating a new build, don't tag it]" \
            "--nowait[Don't wait on build]" \
            "--noinherit[Don't follow inheritance]" \